docs/resources/tensorflow_frontend.md

```diff
@@ -13,8 +13,7 @@ Also, the frontend allows loading TensorFlow models in SavedModel, MetaGraph, an
 The current limitations:
```

Comment:
in the diagram above `Pytorch` -> `PyTorch`

Reply:
Corrected in the https://github.com/openvinotoolkit/openvino/pull/18757/commits/7d32eeb06d69215151e41a455ddcfef19272da4b commit